exporter.go

```diff
@@ -35,9 +37,9 @@ func (e *Exporter) Export(
 ) error {
 	var err error
 
-	meta := metadata.AppImageMetadata{}
```

Comment:
Can we avoid this rename to make rebasing #149 easier?

Reply:
👍  Reverted